### ISSUE TITLE
Rethink about modal animate-in

### ISSUE THREAD

**Issue by nikoloza** [vigour-io/modal#18]
Modal animate-in looks not good in some browsers and devices. Even, some Safari related optimizations brought ugly things for Chrome as well. Need to redo this.

**Comment by jimdebeer** [comment]
maybe we should add a few variants in modal based on device type/speed?

**Comment by nikoloza** [comment]
@jimdebeer yes, check UA and do adequate stuff. That'd be great for many things. For good ones we can go wild :stuck_out_tongue:
